test/cases/engine_internals_test.rb

```diff
@@ -37,7 +37,7 @@ def inspect
         print Haml::Engine.new('%div{ foo: true, bar: false }').render
       HAML
       f.close
-      out = IO.popen([RbConfig.ruby, f.path], &:read)
+      out = IO.popen([RbConfig.ruby, '-W0', f.path], &:read)
```

Comment:
Can you add a documentation note here on why the `-W0` is there and what it's used for?

Reply:
here the I attempted to set ruby interpreter warning level to 0 (=silence), with hopes that jruby warning wouldn't be printed out. I see that you've already merged the PR, I guess, I can leave it as is.